docs/modules/ROOT/pages/configuration/cache-expiration-time.adoc

```diff
@@ -51,7 +51,7 @@ kubectl patch -n kubearchive deployment kubearchive-api-server --patch-file path
 [IMPORTANT]
 .Expiring the cache
 ====
-To expire the current cache restart the `kubearchive-api-server` deployment with
+To expire the current cache, restart the `kubearchive-api-server` deployment with
```

Comment:
I don't think this comma is correct, its splitting a coherent phrase into two. "To expire the current cache" does not have meaning by itself.